### DISCUSSION TITLE
How to debug the client project width serve devtool

### DISCUSSION THREAD

**Discussion by zhengtulymGh** [Budibase/budibase#5466] [Help]
It confuse me that the client package do not have use `sirv-cli `  to start a server and show inside iframe tag in builder package. 
So how to debug the client project width serve devtool?
<img width="1021" alt="v消息20220418-112622" src="https://user-images.githubusercontent.com/13768025/163750141-7e0bf047-8473-430b-bf82-7fe0692fcf4b.png">

**Comment by mjashanks** [comment]
Hi @zhengtulymGh 

It's been a whlie since I wrote any code, but, as far as I know, you will need to debug the client library whilst running an app for real. I.e. do not debug inside the builder preview - run your app and debug from there.
